### ISSUE TITLE
[Translation] I would like to be able to select the Japanese language.

### ISSUE THREAD

**Issue by ToshihiroMakuuchi** [EdgeTX/edgetx#1219]
I recently noticed that there is a language switch for Chinese, which was not a feature in OpenTX.
I would really like to be able to switch to Japanese in the same way.
If you make changes to the following file and provide the JP (Japanese) option, I would like to prepare the 
radio/src/translation/jp.h.txt file. (I'm Japanese).
I would be very grateful for your cooperation.

radio/src/CMakeLists.txt
radio/src/translations.h
radio/src/fonts/CMakeLists.txt
radio/src/gui/colorlcd/layouts/topbar_impl.cpp
radio/src/gui/colorlcd/themes/480_default.cpp
radio/src/gui/colorlcd/themes/flysky.cpp
radio/src/translations/tts_jp.cpp
tools/charset.py

Best Regards.

**Comment by pfeerick** [comment]
Richard is in the middle of redoing the entire mess that is translations, with the goal of making it easier to understand, to remove all hard coded strings, and make it so there should only be one place you need to edit for translations. So once that is done, it would be great we can ping you then to help with the JP language translation. Funnily enough, we actually discussed just last night that this is needed ;)

**Comment by ToshihiroMakuuchi** [comment]
Thank you for your prompt reply.
It would be very nice to have a unified translation environment with [Crowdin](https://crowdin.com/project/betaflight-configurator) similar to Betaflight Configurator.
We Japanese are looking forward to it.

Best Regards.

**Comment by richardclli** [comment]
Yeah, will do Japanese for sure.  Just need a translator, will notify you when we are ready.

**Comment by raphaelcoeffic** [comment]
@ToshihiroMakuuchi are you still interested? 🥹

**Comment by richardclli** [comment]
If you want to work this out, please use the lvgl branch.  And add the translation file for Japanese.

**Comment by ToshihiroMakuuchi** [comment]
@richardclli 
Thank you very much for contacting me. And check the contents.

**Comment by raphaelcoeffic** [comment]
@ToshihiroMakuuchi the first we need is a translation file like all the other `.h` (Check `en.h` For the master file). Just write the translation with Kanji (if possible; otherwise tell me what you need), and we will work out the rest.

**Comment by raphaelcoeffic** [comment]
> @ToshihiroMakuuchi the first we need is a translation file like all the other `.h` (Check `en.h` For the master file). Just write the translation with Kanji (if possible; otherwise tell me what you need), and we will work out the rest.
> 
> the file needs to be encoded with UTF8 to work with tools we use.

If you have free font suggestions, let me know. At the moment we use Google’s Noto CJK font, but others would be possible as well.

**Comment by ToshihiroMakuuchi** [comment]
@raphaelcoeffic
Pull request to lvgl branch.
Japanese file [jp.h] created. Could you please confirm this ?

**Comment by ToshihiroMakuuchi** [comment]
This one is closed because we were able to handle it thanks to all of you.
